docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md

```diff
@@ -39,37 +40,37 @@ $[TIZEN_STUDIO_DIR]/tools/ide/bin$ ./tizen package -t wgt -s [CERTIFICATE_PROFIL
 
     ![Create Web Widget 4](./media/create_web_sample_cs_4.png)
 
-2. Check manifest file of web application
+2. Check the **config.xml** file of the Web application.
 
     ```xml
     <?xml version="1.0" encoding="UTF-8"?>
     <widget xmlns="http://www.w3.org/ns/widgets" xmlns:tizen="http://tizen.org/ns/widgets" id="http://yourdomain/WorldClockWidget" version="1.0.0" viewmodes="maximized">
         <access origin="*" subdomains="true"></access>
         <tizen:app-widget id="TestHybrid.WorldClockWidget.WorldClockWidget" primary="true">
             <tizen:widget-label>World Clock Widget</tizen:widget-label>
-            <tizen:widget-content src="widget/Widget/index.html">
-                <tizen:widget-size preview="widget/Widget/preview.png">2x2</tizen:widget-size>
+            <tizen:widget-content src="widget/WorldClockWidget/index.html">
+                <tizen:widget-size preview="widget/WorldClockWidget/preview.png">2x2</tizen:widget-size>
             </tizen:widget-content>
         </tizen:app-widget>
         <tizen:application id="TestHybrid.WorldClockWidget" package="TestHybrid" required_version="2.3.2"/>
         <content src="index.html"/>
         <feature name="http://tizen.org/feature/screen.size.all"/>
-        <icon src="icon.png"/>
-        <name>WorldClockWidget</name>
-        <tizen:privilege name="http://tizen.org/privilege/application.launch"/>
-        <tizen:profile name="wearable"/>
+		<icon src="icon.png"/>
+		<name>WorldClockWidget</name>
+		<tizen:privilege name="http://tizen.org/privilege/application.launch"/>
+		<tizen:profile name="wearable"/>
     </widget>
     ```
-    package id of web application is set like below :
-     __*package="TestHybrid"*__
+    > [!NOTE]
+    > Ensure that the package ID of the Web application is set to **TestHybrid**. For example, `package="TestHybrid"`.
 
-3. Create .NET Sample Application
+3. Launch the latest version of Visual Studio and create a .NET sample application.
```

Comment:
Please provide the latest images in English for the two images

Reply:
We cannot push commits to a public repository inside the company network, instead we will provide urls for the images.

https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/create-dotnet-hybrid-package-1.png
https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/create-dotnet-hybrid-package-2.png
https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/create-dotnet-hybrid-package-3.png

Reply:
@hakkyu-samsung,

Thank you for the images. I will update.